src/output/mod.rs

```diff
@@ -23,9 +21,7 @@ pub fn log(config: &Config) {
 
     let mut window_id = 0;
 
-    let mut snapshot = Snapshot {
-        prev: HashMap::new(),
-    };
+    let mut snapshot = Snapshot::default();
```

Comment:
Shall we call this `MetricsSnapshot` or `MetricsState` or something to distinguish it from the individual histogram snapshot type?

Reply:
Renamed to `MetricsSnapshot`